modules/ai/assets/js/editor/components/prompt-history/parts/modal-upgrade.js

```diff
@@ -1,24 +1,34 @@
 import { Button, Stack, Typography } from '@elementor/ui';
 import { UpgradeIcon } from '@elementor/icons';
+import { getTranslatedPromptHistoryType, HISTORY_TYPES } from '../history-types';
 
 const VARIANT_FULL = 'full';
 const VARIANT_SMALL = 'small';
 
 const messages = {
-	[ VARIANT_FULL ]: __( 'Want to see your image generation history for as far as the past 90 days?', 'elementor' ),
-	[ VARIANT_SMALL ]: __( 'Want to see your image generation history for the past 90 days?', 'elementor' ),
+	// Translators: %s: History type.
+	[ VARIANT_FULL ]: __( 'Want to see your %s generation history for as far as the past 90 days?', 'elementor' ),
+	// Translators: %s: History type.
+	[ VARIANT_SMALL ]: __( 'Want to see your %s generation history for the past 90 days?', 'elementor' ),
```

Comment:
I suggest you to move the messages to consts/enum file

Reply:
They're actually needed only here, so I think keeping them as constants in the same file should work for us 🙂